matrix/docker/matrix_converter.py

```diff
@@ -13,8 +13,11 @@
 import scipy.io
```

Comment:
Since this PR makes changes to file conversion, it needs to be accompanied with a rebuild of the docker image at https://github.com/HumanCellAtlas/matrix-service/blob/master/docker-images/matrix-converter/Makefile#L20 and increment of image version in use for job definition at https://github.com/HumanCellAtlas/matrix-service/blob/master/terraform/modules/matrix-service/infra/converter_batch.tf#L216